pyanaconda/modules/boss/boss.py

```diff
@@ -38,15 +40,24 @@
 @dbus_interface(DBUS_BOSS_NAME)
 class Boss(BaseModule):
 
-    def __init__(self, module_manager=None):
+    def __init__(self, module_manager=None, install_manager=None):
         super().__init__()
+        self._module_manager = module_manager or ModuleManager()
+        self._install_manager = install_manager or InstallManager()
 
-        if module_manager is None:
-            self._module_manager = ModuleManager()
+    def _setup_install_manager(self):
+        # set modules to the installation manager
+        modules = self._module_manager.running_module_services
```

Comment:
This is really weird, because `running_module_services` is empty at this point. It works only because the managers share the same list of modules (the object).

Reply:
Aha I've missed that, but this must be changed in overall structure. I'm not sure if `ModuleManager` should be passed to the `InstallManager` but I don't like this solution. I think there should be something more general to move modules around.

Reply:
Maybe just add there a comment for now. We need to find a way how to synchronize the boss anyway.

Reply:
Ok, I'll add a comment.